### PR TITLE
feat(fantom): add spiritswap and spookyswap dexs

### PR DESCRIPTION
**Pull request by developerfred**
## Description

Integrations Fantom Dexs:

- SpiritSwap 
- SpookySwap

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
  - [x] I have removed all in development `console.log`s
  - [x] I have removed all commented code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

### CODE REVIEW

**Comment by UncleBill** [comment]
@developerfred Thanks for your work! It looks great.

We don't have `SPIRIT` in our token list for fantom yet, could you help to append it in https://github.com/DimensionDev/Mask-Token-List ?

image

**Comment by developerfred** [comment]
@UncleBill  done here: https://github.com/DimensionDev/Mask-Token-List/pull/57

**Review by UncleBill — DISMISSED** [review]

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Review by Jack-Works — DISMISSED** [review]

**Comment by wenluomask** [comment]
I tried the swap pair FTM->BOO in SpritSwap and SpookySwap, but all failed with RPC error. Could you check what's going on?
Screen Shot 2022-02-08 at 3 22 24 PM
Screen Shot 2022-02-08 at 3 18 17 PM
Screen Shot 2022-02-08 at 3 18 44 PM

**Comment by developerfred** [comment]
@wenluomask I will check here

File: .ecrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "Disable": {
+    "Indentation": true
+  }
+}
```
Comment (UncleBill) [inline .ecrc:5]: What is this file for? If there is no purpose, please remove it.

**Comment by developerfred** [comment]
@wenluomask Now that I've updated the project, can you check if the RPC error still persists?

File: .ecrc (hunk 1 above)
Comment (developerfred) [reply .ecrc:5]: Look -> https://github.com/DimensionDev/Maskbook/commits/develop/.ecrc

Comment (developerfred) [reply .ecrc:5]: Ref: https://github.com/DimensionDev/Maskbook/pull/5610

Comment (UncleBill) [reply .ecrc:5]: I see. Thanks

**Comment by guanbinrui** [comment]
I'm closing this PR. Cause it doesn't belong to any bounty issue.

**Pull request closed without merging**